Sources/SwiftSyntax/BumpPtrAllocator.swift

```diff
@@ -35,6 +37,9 @@ public class BumpPtrAllocator {
   private var customSizeSlabs: [Slab]
   private var _totalBytesAllocated: Int
 
+  /// Construct a new `BumpPtrAllocator` that allocates `slabSize` at a time.
+  /// Once all memory in a slab has been used, it allocates a new slab and no
+  /// memory allocations are necessary until that slab is completely filled up.
```

Comment:
Would it make sense to to have something line 
```
Construct a new ``BumpPtrAllocator``.
```

Just that the class have same documentation and it could start drifting.